include/can/core/message_writer.hpp

```diff
@@ -44,9 +44,9 @@ class MessageWriter {
     }
 
     void set_queue(QueueType* q) { queue = q; }
+    can_ids::NodeId node_id;
```

Comment:
Can we please keep this member private? Having a `set_node_id` makes it clear that nobody is reading this value. Just `set_queue`.

Reply:
okayyyy